.idea/codeStyles/Project.xml

```diff
@@ -157,4 +150,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Don't push changes made into the system files

Reply:
how to make / revert these changes ?

Reply:
For simplicity you can copy of code of that file from here: https://github.com/oppia/oppia-android/tree/develop/.idea

And paste it back in your project and make a commit. Once that is done make sure you go to `Files Changed` section on your PR and see that the file no longer exists.